scipy/stats/_result_classes.py

```diff
@@ -1,9 +1,10 @@
 # This module exists only to allow Sphinx to generate docs
-# for the result objects returned by some functions in stats.
+# for the result objects returned by some functions in stats
+# _without_ adding them to the main stats documentation page.
 
 """
-Result classes
---------------
+Result classes used in :mod:`scipy.stats`
+-----------------------------------------
```

Comment:
Meant to leave this alone.
```suggestion
Result classes
---------------
```